src/components/DayPicker.jsx

```diff
@@ -365,14 +369,19 @@ export default class DayPicker extends React.Component {
   }
 
   onPrevMonthClick(nextFocusedDate, e) {
+    const { translationValueRTL } = this.state;
+    const { isRTL } = this.props;
+
     if (e) e.preventDefault();
 
     if (this.props.onPrevMonthClick) {
       this.props.onPrevMonthClick(e);
     }
 
-    const translationValue =
-      this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
+    let translationValue = this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
```

Comment:
More that this function does:
```
let translationValue = this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
if (isRTL && this.isHorizontal()) {
  translationValue = translationValueRTL - this.dayPickerWidth;
}
```

and `onNextMonthClick` does
```
const convertedTranslationValueRTL = this.dayPickerWidth + this.state.translationValueRTL;
let translationValue = this.props.isRTL ? convertedTranslationValueRTL : -this.dayPickerWidth;

if (this.isVertical()) {
  translationValue = -this.getMonthHeightByIndex(1);
}
```
These two sets of ternaries/conditionals do pretty much the exact same thing, but look totally different. They should look the same. You could accomplish this by changing `onNextMonthClick` to have:
```
let translationValue = this.isVertical() ? -this.getMonthHeightByIndex(1) : -this.dayPickerWidth;
if (isRTL && this.isHorizontal()) {
  translationValue =  this.dayPickerWidth + this.state.translationValueRTL;
}
```